packages/plans-grid-next/src/components/shared/tooltip/index.tsx

```diff
@@ -61,25 +40,28 @@ export const Plans2023Tooltip = ( {
 
 	return (
 		<>
-			<HoverAreaContainer
-				className="plans-2023-tooltip__hover-area-container"
+			<span
+				className="plans-grid-next-tooltip__hover-area-container"
 				ref={ tooltipRef }
 				onMouseEnter={ () => ! isTouch && setActiveTooltipId( id ) }
 				onMouseLeave={ () => ! isTouch && setActiveTooltipId( '' ) }
 				onTouchStart={ () => isTouch && setActiveTooltipId( getMobileActiveTooltip() ) }
 				id={ id }
 			>
 				{ children }
-			</HoverAreaContainer>
-			<StyledTooltip
+			</span>
+			<CoreTooltip
+				className="plans-grid-next-tooltip"
 				isVisible={ isVisible }
 				position="top"
 				context={ tooltipRef.current }
 				hideArrow
 				showOnMobile={ showOnMobile }
 			>
 				{ text }
-			</StyledTooltip>
+			</CoreTooltip>
 		</>
 	);
 };
+
+export default Tooltip;
```

Comment:
It feels so weird that we create the localised Tootlip component and also export it for use outside. I'm not sure what we are trying to achieve that's not possible with Core directly 🤔

Can we investigate and seek to understand? Is it just a wrapper, and is the wrapper used many times in the code (that it needs its own component)?

Reply:
> Can we investigate and seek to understand? Is it just a wrapper, and is the wrapper used many times in the code (that it needs its own component)?

I think never mind. It's obviously used in both grids, maybe elsewhere too. It does signal an abstraction elsewhere may be missing (like a shared "feature" component that would just render the tooltip and not require a Tooltip component instead)